src/cow-react/modules/wallet/web3-react/connection/index.tsx

```diff
@@ -1,7 +1,8 @@
 import { Connector } from '@web3-react/types'
 
 import { isMobile } from 'utils/userAgent'
-import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
+import { ALL_SUPPORTED_CHAIN_IDS } from 'constants/chains'
```

Comment:
Isn't anything like this also exported from the SDK?

Reply:
In the next version